spec.go

```diff
@@ -174,6 +174,103 @@ WRAP:
 	return t.In(origLocation)
 }
 
+// Prev returns the previous time this schedule is activated, less than the given
+// time.  If no time can be found to satisfy the schedule, return the zero time.
+func (s *SpecSchedule) Prev(t time.Time) time.Time {
```

Comment:
Not sure if it was wanted or not, but this function should be added to the Schedule interface for it to be accessible externally.

Reply:
Okay, I'll add it to the interface and to the ConstantDelaySchedule struct.

Reply:
okay, done

Reply:
There hasn't been any contribution to the repo since 2020 so let's hope that someone merges this! Thanks for implementing it though

Reply:
thanks